inc/class-astra-global-palette.php

```diff
@@ -24,12 +24,75 @@ class Astra_Global_Palette {
 	 * @since x.x.x
 	 */
 	public function __construct() {
-		add_action( 'after_setup_theme', array( $this, 'support_editor_color_palette' ) );
+		/**
+		 * Adding backward for theme.json file.
+		 *
+		 * If theme.json is not present in the child theme load Global Color Palette in the editor using add_theme_support( 'editor-color-palette', $editor_palette );.
+		 *
+		 * This is a known issue in Gutenberg - If theme.json is not present in the child theme, it does fallback to the parent theme's theme.json file.
+		 *
+		 * This will be fixed in the future updates of WordPress/Gutenberg.
```

Comment:
```suggestion
		 * If theme.json is not present in the child theme load Global Color Palette in the editor using add_theme_support( 'editor-color-palette', $editor_palette );.
		 * This is a known issue in Gutenberg - If theme.json is not present in the child theme, it does fallback to the parent theme's theme.json file.
		 * This will be fixed in the future updates of WordPress/Gutenberg.
```

Removing extra lines in the comment description.

Reply:
Done!